client/app/src/services/network.service.js

```diff
@@ -131,13 +131,24 @@
       // peer.market={
       //   price: { btc: '0' },
       // }
-      $http.get('http://coinmarketcap.northpole.ro/api/v5/' + network.token + '.json', { timeout: 2000 })
+      $http.get('https://api.coinmarketcap.com/v1/ticker/' + network.token, { timeout: 2000 })
         .then(function (res) {
-          if (res.data.price && res.data.price.btc) {
-            res.data.price.btc = Number(res.data.price.btc).toFixed(8) // store BTC price in satoshi
+          if (res.data[0] && res.data[0].price_btc) {
+            res.data[0].price_btc = Number(res.data[0].price_btc).toFixed(8) // store BTC price in satoshi
           }
-          storageService.set('lastPrice', { market: res.data, date: new Date() }, true)
-          peer.market = res.data
+          storageService.set('lastPrice', { market: res.data[0], date: new Date() }, true)
+          peer.market = res.data[0]
+          $http.get('https://api.fixer.io/latest?base=USD', { timeout: 2000}).then( function (result) {
```

Comment:
Do we care about storing the value in every different one of the currencies below? Or do we just care about what the user has active?

If we just care about what the user has active, then maybe it'd be a good idea to consider passing in the convert param during the API call to coinmarketcap. That'd save us from doing the additional API call to api.fixer.io.

Reply:
I wrote it this way to mimic the old api functionality, which gave us the value in every different currency. We simply store all the currency values and toggle the right one. 

It is a bit wasteful on space since we don't need all the currencies, only the toggled active one. Only downside I could see to that is that we'd have to have a new api call each time the user toggles another currency (which probably will be rarely). 

I'd be down to work on implementing that, just trying to get this change out asap since the old api is frozen and none of the wallets are showing the right price.